README.md

```diff
@@ -262,6 +262,27 @@ $ node example.js 99.3
 { _: ['99.3'] }
 ```
 
+### parse bash ANSI-C strings
+
+* default: `false`
+* key: `parse-bash-ansi-c-strings`
+
+Should arguments that look like [ANSI-C quoted strings](https://www.gnu.org/software/bash/manual/html_node/ANSI_002dC-Quoting.html) (a bash-only feature) be treated as such?
+
+_if disabled:_
+
+```console
+> example.js $'hello\\nworld'
+{ _: [ "$'hello\\nworld'" ] }
```

Comment:
I wrote this example by hand. how do I generate it properly?

Reply:
Easiest thing to do might be to clone `yargs` and `yargs-parser`, then do this:

```
cd yargs-parser
npm link .
cd yargs
npm link yargs-parser
// write your example in a file that includes the local copy of yargs.
```